src/Managed/Scaffolding.repoproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project>
```

Comment:
I guess we check these in because they take a while to generate and won't change very often (basically never in a patch branch)?

Reply:
Correct. Think of these like our resx and Designer.cs files. It's generated, but checked in.